README.md

```diff
@@ -1,12 +1,28 @@
 # incident_io
 
-An Elixir client for the [incident.io API](https://api-docs.incident.io/).
-
 [![Build Status](https://github.com/sgerrand/ex_incident_io/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/sgerrand/ex_incident_io/actions/workflows/ci.yml)
 [![Coverage Status](https://coveralls.io/repos/github/sgerrand/ex_incident_io/badge.svg?branch=main)](https://coveralls.io/github/sgerrand/ex_incident_io?branch=main)
 [![Hex.pm](https://img.shields.io/hexpm/v/incident_io.svg)](https://hex.pm/packages/incident_io)
 [![Documentation](https://img.shields.io/badge/documentation-gray)](https://hexdocs.pm/incident_io/)
 
+An Elixir client for the [incident.io API](https://api-docs.incident.io/).
+
+With just a few lines of code you can begin interacting:
+
+```elixir
+Mix.install([
+  {:incident_io, "~> 0.1"}
+])
+
+client = IncidentIo.Client.new(%{api_key: System.fetch_env!("INCIDENT_API_KEY")
+IncidentIo.IncidentsV2.create(client, idempotency_key: "you-idempotency-key", visibility: :public)
+```
+
+## Requirements
+
+You'll need an [incident.io](https://incident.io/) account to use this package.
+[Sign up on their website](https://incident.io/) if you don't already have one.
```

Comment:
This part is critical for anyone who wants to use this package so wanted to prioritise it.